README.md

```diff
@@ -79,6 +79,10 @@ in the Kubernetes documentation.
 
 Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
-## Slack Channels
+## Contact
 
-Join us at [Rook ceph-csi Channel](https://rook-io.slack.com/messages/CG3HUV94J/details/)
+Please use the following to reach members of the community:
+
+- Slack: Join our [slack channel](https://cephcsi.slack.com)
+- Forums: [rook-dev](https://groups.google.com/forum/#!forum/ceph-csi)
+- Twitter: [@rook_io](https://twitter.com/CephCsi)
```

Comment:
Header text reads"@rook_io" and should read "@cephcsi" instead.

I am no social media expert, but do we actually need a separate twitter channel? Should this be done in conjunction with folks managing the existing Ceph handles and such?

Reply:
Corrected this too. Done.

Reply:
we can have a seperate channel if someone maintains it up to date. `@rook_io` to `@cephcsi`

Reply:
yep, its perfectly fine to have a seperate channel. thanks